internal/events/src/events/__tests__/mi-events.test.ts

```diff
@@ -18,13 +18,15 @@ describe("MI event validations", () => {
     expect(event).toEqual(
       expect.objectContaining({
         type: "uk.nhs.notify.supplier-api.mi.SUBMITTED.v1",
+        plane: "data-plane",
         specversion: "1.0",
         source: "/data-plane/supplier-api/prod/submit-mi",
         id: "8f2c3b44-4e65-5b1b-a678-1f0bf3d4d222",
         time: "2025-11-16T10:30:00.000Z",
         datacontenttype: "application/json",
         dataschema:
           "https://notify.nhs.uk/cloudevents/schemas/supplier-api/mi.SUBMITTED.1.0.0.schema.json",
+        dataschemaversion: "1.0.0",
```

Comment:
This was removed from the envelope profile in favour of the dataschema URL including the same string - is it required for something in particular?